client/blocks/comments/style.scss

```diff
@@ -350,3 +354,12 @@
 		transform: rotate( 180deg );
 	}
 }
+
+.comments__comment-count {
+	color: lighten( $gray, 10% );
+	size: 13px;
```

Comment:
I've set it to 14px as that's what @fraying originally specced in #7423. That's also the same size as the 'view earlier comments' link.